# 自定义消息服务载体
# 客户端请求时发送的两个数字
int32 num1
int32 num2

# 响应部分用3条杠分割
---
# 服务器响应发送的数据
int32 sum